app/views/clients/show.html.slim

```diff
@@ -65,7 +65,7 @@ h1= @client.contact.full_name
 
       tr
         td= t('registered_by')
-        td= link_to @client.user.full_name, user_path(@client.user)
+        td= link_to @client.user.full_name, profile_path(@client.user)
```

Comment:
`if @client.user.profile.present?` and you need to change `profile_path(@client.user)` to `profile_path(@client.user.profile)`, because `@client.user != @client.user.profile`